<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Line" name="lineH_U">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>441</width>
      <height>20</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="midLineWidth">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="lineH_D">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>320</y>
      <width>441</width>
      <height>20</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="midLineWidth">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="lineV_L">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>50</y>
      <width>16</width>
      <height>421</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="midLineWidth">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="lineV_R">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>50</y>
      <width>16</width>
      <height>421</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="midLineWidth">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonU_L">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>141</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonU_C">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>50</y>
      <width>141</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonU_R">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>50</y>
      <width>141</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonM_L">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>190</y>
      <width>141</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonM_C">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>190</y>
      <width>141</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonM_R">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>190</y>
      <width>141</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonD_L">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>330</y>
      <width>141</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonD_C">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>330</y>
      <width>141</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonD_R">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>330</y>
      <width>141</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="turn">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>492</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
